src/Sulu/Bundle/ContentBundle/Controller/NodeController.php

```diff
@@ -582,4 +584,13 @@ public function getSecurityContext()
             return 'sulu.webspaces.' . $webspace->getKey();
         }
     }
+
+    /**
+     * Returns the class name of the object to check
+     * @return string
+     */
+    public function getSecuredClass()
+    {
+        return Structure::class;
```

Comment:
By storage layer you mean doctrine? Then we couple to that, what I would not prefer... What if you don't use doctrine to store this data? That wouldn't work then anymore.